implementation/src/main/java/io/smallrye/config/PropertiesConfigSource.java

```diff
@@ -27,36 +27,41 @@
 
 import org.eclipse.microprofile.config.spi.ConfigSource;
 
+import static io.smallrye.config.utils.ConfigSourceUtil.*;
```

Comment:
Again, not sure if the build was run as we don't want all encompassing static imports